libs/generic-view/ui/src/lib/interactive/form/input/search-input.tsx

```diff
@@ -60,17 +94,45 @@ export const SearchInput: APIFC<FormSearchInputData, FormSearchInputConfig> = ({
           </ClearButton>
         )}
       </InputWrapper>
+      {Children.map(children, (child) => {
+        if (
+          isValidElement(child) &&
+          child.props.componentName === "form.searchInputResults"
+        ) {
+          return cloneElement(child as ReactElement, {
+            componentRef: resultsRef,
+          })
+        }
+        return null
+      })}
     </Wrapper>
   )
 }
 
 export default SearchInput
```

Comment:
```suggestion
```

Reply:
Removed in [c1d1f9a](https://github.com/mudita/mudita-center/pull/2203/commits/c1d1f9adf8d78a3003a7f9ae66d0ec81eb9ec644)